plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -228,6 +251,64 @@ describe('CassandraDriverInstrumentation', () => {
         assertSingleSpan('cassandra-driver.execute', query.substr(0, 25));
       });
     });
+
+    describe('responseHook', () => {
+      after(() => {
+        instrumentation.setConfig({
+          responseHook: () => {},
+        });
+      });
+
+      it('adds custom attributes to span', async () => {
+        const responseAttributeName = 'response.attribute';
+        const customAttributeName = 'custom.attribute';
+        const customAttributeValue = 'custom attribute value';
+
+        instrumentation.setConfig({
+          responseHook: (span: Span, responseInfo: ResponseInfo) => {
+            const row = responseInfo.response.first();
+            span.setAttribute(responseAttributeName, row.count.toString());
+            span.setAttribute(customAttributeName, customAttributeValue);
+          },
+        });
+
+        await client.execute(
+          "SELECT count(*) FROM system_schema.columns WHERE keyspace_name = 'ot' AND table_name = 'test';"
+        );
+
+        assertAttributeInSpan('cassandra-driver.execute', [
+          {
+            attributeName: customAttributeName,
+            attributeValue: customAttributeValue,
+          },
+          {
+            attributeName: responseAttributeName,
+            attributeValue: '2',
+          },
+        ]);
```

Comment:
There is already an [`Attributes` interface](https://github.com/open-telemetry/opentelemetry-js/blob/afe065772360d3acec8b8fc15e20536fd896c6c6/api/src/common/Attributes.ts#L22) that does exactly this.

Do we need to introduce a new data structure?

An alternative for the attributes array can be:
```js
{[customAttributeName]: customAttributeValue, [responseAttributeName]: '2'}
```

Reply:
Thanks! wasn't aware of that interface. Using it now instead :)